src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
```

Comment:
There should not be a redundant empty line after the class declaration. Please remove the empty line to adhere to the coding standards outlined in the checklist.